client/scss/components/_scroll_to_info.scss

```diff
@@ -0,0 +1,18 @@
+@keyframes scrollToWobble {
```

Comment:
Good point – I wanted to keep animation names namespaced by component to prevent clashing. I've updated is to be less ambiguous.